src/twisted/conch/test/keydata.py

```diff
@@ -84,6 +85,16 @@
   'curve': b'ecdsa-sha2-nistp521'
 }
 
+ED25519Data = {
```

Comment:
I realize that this is just repeating an instance of an existing pattern, but could we replace this with something that generates a test key on the fly, rather than supplying static data?  Long term, we should be deleting every static private key in this module.